t/lib/OpenQA/Test/Utils.pm

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2016-2020 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+
+=head2 OpenQA::Test::Utils
+
+Utility functions for tests
+
+=cut
+
+package OpenQA::Test::Utils;
+use strict;
+use warnings;
+
+use Test::Output ();
+use Test::Warnings ();
+use Test::More ();
+
+use base 'Exporter';
+our @EXPORT_OK = qw(&stderr_like &combined_like);
+
+sub stderr_like {
+    my ($code, $re, $label) = @_;
+    my @warnings = Test::Warnings::warnings(sub {
+        Test::Output::stderr_like(sub { $code->() }, $re, $label);
+    });
+    if (@warnings) {
+        is(scalar @warnings, 0, 'Got no unexpected warnings in stderr_like call')
+          or Test::More::diag "Warnings: " . join '', @warnings;
+    }
+}
+
+sub combined_like {
+}
```

Comment:
Seems like this is still unimplemented. If we currently don't need/use it I would rather remove the empty definition.

Reply:
I was just experimenting. I don't rely on the tests passing locally.